libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -453,6 +453,10 @@ const AP_Param::GroupInfo AP_InertialSensor::var_info[] = {
     // @Bitmask: 0:FirstIMU,1:SecondIMU,2:ThirdIMU
     AP_GROUPINFO("ENABLE_MASK",  40, AP_InertialSensor, _enable_mask, 0x7F),
 
+    // @Group: DNTCH_
+    // @Path: ../Filter/HarmonicNotchFilter.cpp
+    AP_SUBGROUPINFO(_harmonic_notch_filter, "DNTCH_",  41, AP_InertialSensor, HarmonicNotchFilterParams),
```

Comment:
Maybe - ultimately we want to wire this up to the throttle output so that the center frequency varies dynamically. Happy to go with whatever the consensus is here. Originally the class was called DynamicNotchFilter.

Reply:
> Originally the class was called DynamicNotchFilter.

Yeah, that's why I thought this could be changed too. I don't really have a strong opinion on either. @lthall?

Reply:
I would say go with HNTCH

Reply:
Fixed